2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,35 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
```

Comment:
It might be best to make sure that the todo list `<ul>` element is empty here before we fill it out - this means that we can call the same function again to build the list of todos no matter what the value of the todos is.